xsd/netex_part_3/part3_salesTransactions/netex_customerPurchasePackage_version.xsd

```diff
@@ -599,6 +634,18 @@ Rail transport, Roads and Road transport
 			</xsd:element>
 		</xsd:sequence>
 	</xsd:group>
+	<xsd:group name="CustomerPurchasePackageHistoryGroup">
+		<xsd:annotation>
+			<xsd:documentation>FARE CONTRACT entries CUSTOMER PURCHASE PACKAGE.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element name="customerPurchasePackageEntries" type="fareContractEntries_RelStructure" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>FARE CONTRACT ENTRIES relating to CUSTOMER PURCHdASE PACKAGE. May be references to ENTRIEs held,  or entry instances inlined here  for convenience so that , they may exchanged  along with package.  This can include any type of entry:  note that TRAVEL SPECIFICATIONs and SALES TRANSACTIONS may be referenced separately by a gven PACKAGE. +v2.0</xsd:documentation>
```

Comment:
```suggestion
					<xsd:documentation>FARE CONTRACT ENTRIES relating to CUSTOMER PURCHASE PACKAGE. May be references to ENTRIEs held, or entry instances inlined here  for convenience so that , they may exchanged  along with package.  This can include any type of entry:  note that TRAVEL SPECIFICATIONs and SALES TRANSACTIONS may be referenced separately by a given PACKAGE. +v2.0</xsd:documentation>
```